src/components/pages/OnboardingServiceProvider/OnboardingServiceProvider.tsx

```diff
@@ -172,13 +169,24 @@ const OnboardingServiceProvider = () => {
                     'content.onboardingServiceProvider.callbackUrl.name'
                   )}
                   value={data?.callbackUrl}
-                  validate={(expr) => isWellknownMetadata(expr)}
+                  validate={(expr) => isURL(expr)}
```

Comment:
```expr.endsWith('.well-known/openid-configuration')``` this condition is removed in the updated code. I do not see the corresponding description in the PR details. Is it intentional?

Reply:
@manojava-gk  thanks for asking. Yes it was intentional and description is updated now. Thanks